Lens/Lens.csproj

```diff
@@ -1,281 +1,72 @@
-    <Compile Include="SyntaxTree\Literals\UnitNode.cs" />
-    <Compile Include="SyntaxTree\NodeBase.cs" />
-    <Compile Include="SyntaxTree\Operators\Binary\BinaryOperatorNodeBase.cs" />
-    <Compile Include="Translations\LexerMessages.Designer.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>LexerMessages.resx</DependentUpon>
-    </Compile>
-    <Compile Include="Translations\ParserMessages.Designer.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>ParserMessages.resx</DependentUpon>
-    </Compile>
-    <Compile Include="Utils\Annotations.cs" />
-    <Compile Include="SyntaxTree\LocationEntity.cs" />
-    <Compile Include="Compiler\TypeSignature.cs" />
-    <Compile Include="Utils\EnumerableExtensions.cs" />
-    <Compile Include="Resolver\FunctionalHelper.cs" />
-    <Compile Include="Utils\HashList.cs" />
-    <Compile Include="SyntaxTree\LexemLocation.cs" />
-    <Compile Include="Resolver\MethodLookupResult.cs" />
-    <Compile Include="Compiler\Local.cs" />
-    <Compile Include="Utils\NodeChild.cs" />
-    <Compile Include="Resolver\TypeExtensions.cs" />
-    <Compile Include="Utils\NodeExtensions.cs" />
-    <Compile Include="Utils\PatternNameBinding.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="Grammar\Grammar.v2.txt" />
-  </ItemGroup>
-  <ItemGroup>
-    <EmbeddedResource Include="Translations\CompilerMessages.resx">
-      <Generator>PublicResXFileCodeGenerator</Generator>
-      <LastGenOutput>CompilerMessages.Designer.cs</LastGenOutput>
-      <SubType>Designer</SubType>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Translations\CompilerMessages.ru.resx" />
-    <EmbeddedResource Include="Translations\LexerMessages.resx">
-      <Generator>ResXFileCodeGenerator</Generator>
-      <LastGenOutput>LexerMessages.Designer.cs</LastGenOutput>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Translations\LexerMessages.ru.resx" />
-    <EmbeddedResource Include="Translations\ParserMessages.resx">
-      <Generator>ResXFileCodeGenerator</Generator>
-      <LastGenOutput>ParserMessages.Designer.cs</LastGenOutput>
-      <SubType>Designer</SubType>
-    </EmbeddedResource>
-    <EmbeddedResource Include="Translations\ParserMessages.ru.resx" />
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="Lens.nuspec" />
-  </ItemGroup>
-  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <!-- To modify your build process, add your task inside one of the targets below and uncomment it. 
-       Other similar extension points exist, see Microsoft.Common.targets.
-  <Target Name="BeforeBuild">
-  </Target>
-  <Target Name="AfterBuild">
-  </Target>
-  -->
-</Project>
\ No newline at end of file
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>net45;net451;net452;net46;net461;net462;net47;netcoreapp2.0</TargetFrameworks>
```

Comment:
Why do we need anything but `net45;netcoreapp2.0` here? Are we using any additional benefits of newer frameworks?